comms/dht/src/test_utils/makers.rs

```diff
@@ -75,14 +75,28 @@ pub fn make_dht_header(
     flags: DhtMessageFlags,
     include_origin: bool,
     trace: MessageTag,
+    include_destination: bool,
```

Comment:
lil clippy warning here

Reply:
damnit I forget to check the tests locally